proposals/description/fip_61.ts

```diff
@@ -0,0 +1,87 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_61: ProposalDescription = {
+  title: 'FIP-61: Maintenance and Peg Updates',
+  commands: [
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'swapDeposit(address,address)',
+      arguments: ['{staticPcvDepositWrapper2}', '{namedStaticPCVDepositWrapper}'],
+      description: 'Swap static deposit out for named static deposit'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeMinter(address)',
+      arguments: ['{daiBondingCurve}'],
+      description: 'Deprecate DAI bonding curve'
+    },
+    {
+      target: 'daiBondingCurve',
+      values: '0',
+      method: 'pause()', // TODO look for other deprecated contracts that need to be paused
+      arguments: [],
+      description: 'Deprecate DAI bonding curve'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeMinter(address)',
+      arguments: ['{tribeRagequit}'],
+      description: 'Deprecate TRIBE ragequit'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: ['0x23970cab3799b6876df4319661e6c03cc45bd59628799d92e988dd8cbdc90e31', '{tribalChiefSyncV2}'],
+      description: 'Add TribalChiefSyncV2'
+    },
+    {
+      target: 'optimisticTimelock',
+      values: '0',
+      method: 'becomeAdmin()',
+      arguments: [],
+      description: 'Become timelock admin'
+    },
+    {
+      target: 'optimisticTimelock',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: ['0xd8aa0f3194971a2a116679f7c2090f6939c8d4e01a2a8d7e41d55e5351469e63', '{tribalChiefSyncV2}'],
```

Comment:
would be nice to be able to do substitution for these roles so instead of putting illegible 64 char string, we just put '{EXECUTOR_ROLE}' and under the hood it gets replaced for the corresponding bytes32

Reply:
Make a ticket